### ISSUE TITLE
Turning mathematical formulae into plain text (searchability and <title> element)

### ISSUE THREAD

**Issue by skalee** [geolexica/geolexica-server#112]
Mathematical formulas can appear in designations, for example:

- function of _n_ variables (IEV 103-01-06)
- _p_-fractile (IEV 103-08-14)
- _∅_ (ISO TC211 2158) — actually it's abbreviation entry, but eventually should be promoted to symbolic designation

This raises at least two following problems:

- MathML cannot appear inside of `<title>` element
- MathML is not properly searchable by our site search

We need to find the best way to turn mathematical formulas to raw text, so that they can appear in `<title>` element and that they are searchable. Perhaps converting MathML to AsciiMath will suit our needs. In fact, in all three above cases the result will be perfect, i.e. `function of n variables`, `p-fractile`, `∅`. That said, I'm not sure if this will hold true for more complicated formulas, or for non-latin scripts.

### TODO

- [x] Decide what to do
- [x] Create an issue for each feature to fix

**Comment by ronaldtse** [comment]
@skalee Let's do AsciiMath for the title element then. Don't seem there is an alternative here.

**Comment by skalee** [comment]
And regarding search, let's also do AsciiMath for now. We will see if this is enough, and imporve later, if needed.

**Comment by skalee** [comment]
We need MathML first, hence it's blocked by #108.

**Comment by skalee** [comment]
Closing in favour of specific issues (#118, #119).
